plugin/src/main/kotlin/com/ritense/plugin/web/rest/PluginConfigurationResource.kt

```diff
@@ -42,12 +41,26 @@ class PluginConfigurationResource(
 ) {
 
     @GetMapping(value = ["/configuration"], produces = [MediaType.APPLICATION_JSON_VALUE])
-    fun getPluginDefinitions(@RequestParam("category") category: String?): ResponseEntity<List<PluginConfigurationDto>> {
+    fun getPluginDefinitions(@RequestParam("category") category: String?,
+                             @RequestParam("includeActionless", defaultValue = "true") includeActionless: Boolean)
```

Comment:
i think this should be a filter on the type of action instead

Reply:
Agreed, when a service task is clicked, we should only show configurations with actions that work for service tasks.